src/goto-programs/remove_function_pointers.cpp

```diff
@@ -80,6 +81,27 @@ class remove_function_pointerst
       compute_address_taken_functions(s.second.value, address_taken);
   }
 
+private:
+  typedef std::list<exprt> functionst;
+
+  bool try_get_precise_call(
+    const exprt &expr, exprt &out_function);
+
+  bool try_get_from_address_of(
+    const exprt &expr, functionst &out_functions);
+
+  bool try_get_call_from_symbol(
+    const exprt &symbol_expr, functionst &out_functions);
+
+  bool try_get_call_from_index(
+    const exprt &index_expr, functionst &out_functions);
+
+  bool try_evaluate_index_value(
+    const exprt &index_value_expr, mp_integer &out_array_index);
+
+  bool try_get_array_from_index(
+    const index_exprt &index_expr, array_exprt &out_array_expr);
+
```

Comment:
I think this changed as I thought it would require that but @peterschrammel has told me that if it fits on one line, then that's OK, see this:

> put each argument on a separate line *if they do not fit after one line break*

https://github.com/diffblue/cbmc/blob/master/CODING_STANDARD#L12